model/task/task_test.go

```diff
@@ -3541,16 +3541,20 @@ func TestArchive(t *testing.T) {
 func TestGetTaskStatsByVersion(t *testing.T) {
 	assert.NoError(t, db.ClearCollections(Collection))
 	t1 := Task{
-		Id:        "t1",
-		Version:   "v1",
-		Execution: 0,
-		Status:    evergreen.TaskSucceeded,
+		Id:               "t1",
+		Version:          "v1",
+		Execution:        0,
+		Status:           evergreen.TaskStarted,
+		ExpectedDuration: 1000000,
+		StartTime:        time.Date(2009, time.November, 10, 12, 0, 0, 0, time.UTC),
 	}
 	t2 := Task{
-		Id:        "t2",
-		Version:   "v1",
-		Execution: 0,
-		Status:    evergreen.TaskFailed,
+		Id:               "t2",
+		Version:          "v1",
+		Execution:        0,
+		Status:           evergreen.TaskStarted,
+		ExpectedDuration: 9000000000000,
```

Comment:
nit. time.(SomeUnit) * some number instead of a random arbitrarily large number.